package.json

```diff
@@ -37,37 +39,40 @@
 		"react-router": "^6.0.0-alpha.1",
 		"react-router-dom": "^6.0.0-alpha.1",
 		"react-tracking": "^7.3.0",
-		"resolve": "1.15.0"
+		"resolve": "1.15.0",
+		"wait-on": "^7.2.0"
 	},
 	"engines": {
-		"node": ">=16.0.0 <17.0.0",
-		"npm": ">=8.0.0 <9.0.0"
+		"node": ">=18.0.0 <21.0.0",
```

Comment:
Since `lts/iron` is what's stated in `.nvmrc` shouldn't the range here be more targeted to v20? `>=18.0.0`?